projects/js-packages/publicize-components/src/components/admin-page/header/index.js

```diff
@@ -77,7 +80,7 @@ const Header = () => {
 						</Button>
 					</div>
 				</Col>
-				{ ! is_wpcom ? (
+				{ isShareLimitEnabled() ? (
```

Comment:
I don't like bringing back this TBH.

Why not just create a utility function like `is_wpcom_simple` and use it in all the places in this PR?

Reply:
I understand that, but what determines that data being available is if the share limits were enabled for the site. VIP don't have them for example.

Reply:
I suppose we could always rename it to `isSharesDataAvailable` or something?